src/components/Loading/spinners/CircleSpinner.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
```

Comment:
From talking with @valentijnnieman some of these seem to be inspired from MIT licensed sources, I think we should add attribution to the code if only for future devs to know where they come from and for follow up purposes.

Reply:
Yes, good call. Completely forgot - most of them come from https://github.com/plotly/dash-core-components/pull/352#issuecomment-437483287 so we should credit them. Just as comments in the source code?

Reply:
I'm fine with code comments except if we do explicit attributions normally through a Notice.txt or similar